servers/rendering/rendering_device_binds.cpp

```diff
@@ -102,22 +99,27 @@ Error RDShaderFile::parse_versions_from_text(const String &p_text, const String
 		if (reading_versions) {
 			String l = line.strip_edges();
 			if (l != "") {
-				int eqpos = l.find("=");
-				if (eqpos == -1) {
-					base_error = "Version syntax is version=\"<defines with C escaping>\".";
+				if (l.find("=") == -1) {
+					base_error = "Missing `=` in '" + l + "'. Version syntax is `version = \"<defines with C escaping>\";`.";
+					break;
+				}
+				if (l.find(";") != -1) {
+					// We don't require a semicolon per se, but it's needed for clang-format to handle things properly.
+					base_error = "Missing `;` in '" + l + "'. Version syntax is `version = \"<defines with C escaping>\";`.";
 					break;
 				}
-				String version = l.get_slice("=", 0).strip_edges();
+				Vector<String> slices = l.get_slice(";", 0).split("=");
```

Comment:
As mentioned in the commit, this will silently ignore anything to the right of `;`, so if we put two defines on the same line by mistake, it will discard the second one silently.

I can make it more robust by throwing an error in `l.split(";").size() > 1`.

Reply:
As this is not GLSL code and will only be used in very specific shader headers by @reduz, I guess it's fine as is. We'll change our mind if we ever waste some time debugging why a define was not used in the generated shader code :P